core/src/main/scala/ai/h2o/sparkling/backend/SharedBackendConf.scala

```diff
@@ -476,6 +482,13 @@ object SharedBackendConf {
       |setLdapLoginDisabled()""".stripMargin,
     "Enable LDAP login.")
 
+  val PROP_LDAP_HASH_LOGIN: BooleanOption = (
+    "spark.ext.h2o.ldaphash.login",
```

Comment:
Instead of introducing a property dedicated just to LDAP, I would propose to create a generic one which use hash authentication on h2o nodes and the main auth mechanism on proxy. (`spark.ext.h2o.proxy.login.only` or better name) The same requirement for doing hash auth on h2o nodes could occur with the combination with Kerberos and with OpenId in future.